fastapi/utils.py

```diff
@@ -19,7 +19,7 @@
 
 
 def is_body_allowed_for_status_code(status_code: Union[int, str, None]) -> bool:
-    if status_code is None:
+    if status_code is None or status_code == "default":
```

Comment:
This wont fix the OpenAPI status code range definitions, right?

See: https://swagger.io/docs/specification/describing-responses/#status-codes

Reply:
That is correct, that is currently not supported. Was that working prior to 0.79? The fix would be easy though:

`if status_code is None or status_code == "default" or status_code in ["1XX", "2XX", "3XX", "4XX", "5XX"]:`

Reply:
> Was that working prior to 0.79?

Yes that was working previously. So eg defining:
```
FastAPI(..., responses={"5XX": {"model": ServerErrorResponse, "description": "Unknown Server Error"}})
```
Yields spec like in https://swagger.io/docs/specification/describing-responses/#status-codes for all of the routes.

Probably these unit tests should be extended to cover the regression: https://github.com/tiangolo/fastapi/blob/master/tests/test_additional_responses_router.py

Reply:
i'd agree, I'll update the PR when i have time (propably tomorrow). Didn't know about the ranges, did you open an issue about those? I'll reference them in this PR for closure then.